Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/ToolTipBackgroundMesh.cs

```diff
@@ -10,16 +10,14 @@ namespace Microsoft.MixedReality.Toolkit.SDK.UX.ToolTips
     /// </summary>
     public class ToolTipBackgroundMesh : ToolTipBackground
     {
-        /// <summary>
-        /// Transform that scale and offset will be applied to.
-        /// </summary>
         [SerializeField]
         [Tooltip("Transform that scale and offset will be applied to.")]
         private Transform backgroundTransform;
 
         private float depth = 1f;
 
-        protected Bounds localContentBounds;
+        // TODO: Unused?
```

Comment:
I did not identify a purpose for this myself when porting it over from HoloToolkit.

I assume it was for something like getting the background's coverage area/hitting area.
I didn't think much of it and didn't remove it. 

I think it should 
1. Pruned 
2. Exposed as public in case we can think of a good case for an external class to reference it.
3. Left as a comment with explanation for 'If a developer wanted to do this, this is the basis of the math'. I think this is the weakest choice because it isn't particularly hard to figure it out or for us to extend the support to do that.

Reply:
If anything is was likely an internal usage thing. I think it can be safely pruned for now.